mobile-widgets/qml/Settings.qml

```diff
@@ -139,201 +139,221 @@ TemplatePage {
 			title: qsTr("Theme")
 
 			GridLayout {
-				id: themeSettings
 				visible: sectionTheme.isExpanded
-				columns: 3
+				columns: 2
 
-				TemplateLabel {
-					text: qsTr("Blue")
-					rightPadding: Kirigami.Units.gridUnit
-					Layout.preferredWidth: gridWidth * 0.15
+				TemplateComboBox {
+					editable: false
+					Layout.columnSpan: 2
+					currentIndex: (ThemeNew.currentTheme === "Blue") ? 0 :
+								  (ThemeNew.currentTheme === "Pink") ? 1 : 2
+					model: ListModel {
+						   ListElement {text: qsTr("Blue")}
+						   ListElement {text: qsTr("Pink")}
+						   ListElement {text: qsTr("Dark")}
+					}
+					onActivated:  {
+						ThemeNew.currentTheme = currentIndex === 0 ? "Blue" :
+												currentIndex === 1 ? "Pink" : "Dark"
```

Comment:
odd whitespace

Reply:
line 149 is alignment (at least in Qt Creator)
line 151-153 is also alignment

Which I thought to be ok (at least you have corrected me, for changing that earlier).